crates/type/src/digest/digests.rs

```diff
@@ -157,6 +157,7 @@ where
         &CONTENT_DIGEST
     }
 
+    #[allow(single_use_lifetimes)]
```

Comment:
I think this should be unnecessary? Can you remove the `'i` lifetime?

Reply:
No, because it's required by the trait